test/e2e/performanceprofile/functests/7_performance_kubelet_node/kubelet.go

```diff
@@ -45,33 +52,38 @@ var _ = Describe("[ref_id: 45487][performance]additional kubelet arguments", Ord
 		profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
 		Expect(err).ToNot(HaveOccurred())
 
-		performanceMCP, err = mcps.GetByProfile(profile)
-		Expect(err).ToNot(HaveOccurred())
+		if !hypershift.IsHypershiftCluster() {
```

Comment:
You can use a `GetPoolName` function hide the details about the platform and handle both cases